kmip/pie/objects.py

```diff
@@ -201,12 +206,13 @@ class SymmetricKey(Key):
         cryptographic_algorithm: The type of algorithm for the SymmetricKey.
         cryptographic_length: The length in bits of the SymmetricKey value.
         value: The bytes of the SymmetricKey.
+        format_type: The format of the key value.
         cryptographic_usage_masks: The list of usage mask flags for
             SymmetricKey application.
         names: The string names of the SymmetricKey.
     """
 
-    def __init__(self, algorithm, length, value, masks=None,
+    def __init__(self, algorithm, length, value, format_type, masks=None,
```

Comment:
That's a good point. I don't think anything else would apply to symmetric keys except for raw. Maybe opaque?

The only format type used with symmetric keys in the testing documentation is raw, so let's stick with that.